gems/smithy-client/spec/smithy-client/waiters_spec.rb

```diff
@@ -0,0 +1,1661 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+module Smithy
+  module Client
+    module Waiters
+      describe Waiters do
+        let(:shapes) do
```

Comment:
I was hoping this would be slimmed down. We don't need to define all of the possibilities up front and we certainly don't need documentation traits etc. This model would be difficult to maintain.

I think we would want context blocks and make small services that have each concrete example.

Let's talk offline.

Reply:
These shapes is a lot to scroll through - might be better to put these in a separate json file?